.github/workflows/ci-rocksdb-build.yml

```diff
@@ -1,7 +1,7 @@
 name: Continuous Integration (Rocksdb Build)
 
 env:
-  ROCKSDB_VERSION: v8.10.0
+  ROCKSDB_VERSION: v9.3.1
```

Comment:
Is 9.3.1 a typo or is 9.2.1 the typo?

Reply:
ahh ty RocksDB `v9.3.1` should be the correct one that matches [`github.com/linxGnu/grocksdb@v1.9.3`](https://github.com/linxGnu/grocksdb/releases/tag/v1.9.3) which is the one used by kava as listed in `go.sum`

Got the rocksdb `v9.2.1` from  cometbft-db that depends on [`github.com/linxGnu/grocksdb@v1.9.2`](https://github.com/linxGnu/grocksdb/releases/tag/v1.9.2)